scripts/docker/clean-logs.sh

```diff
@@ -21,19 +21,30 @@ set -euo pipefail
 
 readonly DIRECTORY="${AIRFLOW_HOME:-/usr/local/airflow}"
 readonly RETENTION="${AIRFLOW__LOG_RETENTION_DAYS:-15}"
+readonly DELETE_LOGS_FOLDERS="${AIRFLOW__LOG_DELETE_FOLDERS:-true}"
```

Comment:
I almost feel like this should just happen regardless and not be able to be turned off. @potiuk, thoughts?